pages/k8s/cdk-addons.md

```diff
@@ -15,10 +15,19 @@ toc: False
 
 In addition to the components strictly required to create a Kubernetes cluster,
 **Charmed Kubernetes** installs and configures some components for convenience
-and/or to add support for specific features.
+and/or to add support for specific features. These 'addons' have historically 
+been 'baked in' with the default install of Charmed Kubernetes, but as these
+components are no longer going to be updated upstream, these are being replaced by 
+specific **Operator Charms**. Such charms are deployed by Juju into Kubernetes
+itself and require some additional steps. 
 
-This page details these components, their purpose and how they may be
-enabled/disabled where applicable.
+<div class="p-notification--positive is-inline">
```

Comment:
N.B. this doesn't display correctly in our preview due to changes in vanilla used in u.c. I will try and sync these at a future date